boards/BMS/Src/Io/Io_CurrentSense.c

```diff
@@ -1,13 +1,30 @@
 #include <stddef.h>
+#include <math.h>
 #include "Io_CurrentSense.h"
 
-ExitCode Io_CurrentSense_ConvertToLowResolutionMainCurrent(float adc_voltage, float *low_res_main_current)
+// Offset voltage of output 1. Found to be 2.45V through testing
+#define OUTPUT_1_OFFSET (2.45f)
+// Sensitivity of output 1: 40mV/A
+#define OUTPUT_1_SENSITIVITY (40e-3f)
+// Voltage divider between sensor output and adc input.
+#define OUTPUT_1_DIV (2.155f / (1.1f + 2.155f))
```

Comment:
`OUTPUT_1_DIV` is the same as `OUTPUT_2_DIV`. Can we use condense them into a single #define?

Reply:
From a firmware perspective that would make sense. But there is actually two different hardware circuits that may change in the future. That is my reason for keeping them separate.

Reply:
sounds good w me